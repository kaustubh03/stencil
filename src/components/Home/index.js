import React, { Component } from "react";
import styles from "./index.module.css";
import NavComponent from "../Navbar";

class Home extends Component {
  render() {
    return (
      <div className={styles.App}>
        <header className={styles.App_header}>
          <div>
            <h1>React Boilerplate</h1>
            <h1>React Boilerplate</h1>
            <h1>React Boilerplate</h1>
          </div>
          <br />
          <div className={styles.description}>
            <h6>
              With CRAv2, Redux, Hygen, Storybook, React Bootstrap, React Icons
            </h6>
          </div>

          <NavComponent />
        </header>
      </div>
    );
  }
}

export default Home;
