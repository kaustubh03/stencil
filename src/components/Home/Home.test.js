import React from 'react'
import renderer from 'react-test-renderer'
import Home from '.'

it('Home: default', () => {
  const component = renderer.create(<Home />)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})