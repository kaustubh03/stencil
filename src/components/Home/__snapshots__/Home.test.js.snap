// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home: default 1`] = `
<div
  className="App"
>
  <header
    className="App_header"
  >
    <div>
      <h1>
        React Boilerplate
      </h1>
      <h1>
        React Boilerplate
      </h1>
      <h1>
        React Boilerplate
      </h1>
    </div>
    <br />
    <div
      className="description"
    >
      <h6>
        With CRAv2, Redux, Hygen, Storybook, React Bootstrap, React Icons
      </h6>
    </div>
    <ul
      className="nav nav-pills"
      role={null}
    >
      <li
        className="btn"
        role="presentation"
      >
        <a
          href="/crud"
          onClick={[Function]}
          onKeyDown={[Function]}
          title="CRUD Example"
        >
          CRUD Example
        </a>
      </li>
      <li
        className="btn"
        role="presentation"
      >
        <a
          href="/api"
          onClick={[Function]}
          onKeyDown={[Function]}
          title="API Calls Using Axios"
        >
          API Example
        </a>
      </li>
      <li
        className="btn"
        role="presentation"
      >
        <a
          href="/react-icons"
          onClick={[Function]}
          onKeyDown={[Function]}
        >
          React Icons
        </a>
      </li>
    </ul>
  </header>
</div>
`;
